academy/0.0-B9lab-Blockchains/2_public.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "Blockchain History"
+order: 3
+description: A Brief History of Blockchain Technology
+tag: fast-track
+---
+
+
+# A Brief History of Blockchain
+
+To fully appreciate Cosmos, it is helpful to understand where blockchain comes from, advances that have unfolded and intractable challenges of other platforms that Cosmos decisively solves. 
+
+Important groundwork was laid in the 1980's and 1990's for what would later be known as blockchain technology. Although the technology itself was not created until 2008, researchers from different fields in computer science and cryptography proposed solutions to many problems of security, transparency, and trust. All of them contributed to the development of the first large scale, successful public blockchain network, Bitcoin.
+
+<!-- TODO: Revise Timeline with Tendermint, 2014 --->
+
+![](images/timeline.png)
+
+The Tendermint Consensus solution was announced in 2014.
```

Comment:
I would include more information here if you already want to talk about Tendermint. But in my review, I moved the image around a bit to a later point because I believe it fits more seamlessly.